content/en/docs/setup/production-environment/index.md

```diff
@@ -0,0 +1,295 @@
+requests that can come in through the API server include
+[Webhook Token Authentication](https://kubernetes.io/docs/reference/access-authn-authz/authentication/#webhook-token-authentication).
+Webhooks and other special authorization types need to be enabled by adding
+[Admission Controllers](https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/)
+to the API server.
+
+## Production workload resources
+
+When planning a production Kubernetes cluster, you need to prepare for the scale and nature of your workloads. Demands from workloads can cause pressure outside of the Kubernetes control plane as well. Here are a few things to consider when managing workload resources:
+
+- *Set namespace limits*: To manage defaults and the limits of user workloads,
+you can set per-namespace quotas on things like memory and CPU. See
+[Manage Memory, CPU, and API Resources](https://kubernetes.io/docs/tasks/administer-cluster/manage-resources/)
+for details. You can also set
+[Hierarchical Namespaces](https://kubernetes.io/blog/2020/08/14/introducing-hierarchical-namespaces/)
+for inheriting limits.
+- *Consider horizontal pod autoscaling*: In preparation for demands on workloads in the
+cluster, you can use pod autoscaling to automatically add additional pods as needed.
+New pods are created based on an observation that a pod is exceeding its CPU
+utilization or other metric provided by the application. See
+[Horizontal Pod Autoscaler](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/)
+for details.
+- *Prepare for DNS demand*: An extreme number of requests coming into the API
+server can cause excessive traffic to the DNS service. If workloads are massively
+scaling up, you will need to make sure your DNS service can scale up as well. See
+[Autoscale the DNS service in a Cluster](https://kubernetes.io/docs/tasks/administer-cluster/dns-horizontal-autoscaling/).
+- *Create additional service accounts*: While user accounts determine what users
+can do on a cluster, a service account defines pod access within a particular
+namespace in the cluster. Unless defined otherwise, a pod takes on the default
+service account for the namespace in which it runs. You can create a new service
+account, containing additional features, that you can add to a new pod to give
+it additional capabilities (see
+[Managing Service Accounts](https://kubernetes.io/docs/reference/access-authn-authz/service-accounts-admin/)).
+For example, you might want to:
+  - Add secrets that a pod could use to pull images from a particular container registry. See [Configure Service Accounts for Pods](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/) for an example.
+  - Assign RBAC permissions to a service account. See [ServiceAccount permissions](https://kubernetes.io/docs/reference/access-authn-authz/rbac/#service-account-permissions) for details.
```

Comment:
Can we link to a different page that signposts readers to these things? They are important, but I don't see them as cluster setup tasks.

Reply:
I'm inclined to leave this, but perhaps I can be persuaded to change it. I'm thinking a production cluster is, by its nature, something you prepare to have a long shelf life as you are setting it up. Part of setting up a production cluster is acknowledging that you are taking on management tasks as well. A Learning environment is something you more typically set up and throw away when you are done. For production, the initial setup might mean preparing it for multi-user use and on-going maintenance.

Reply:
That makes sense - you do need to define access control when you set up a production cluster.

If keeping it, then: can we say the same thing more concisely?

Reply:
Sure. I can do that. Thanks!

Reply:
Done.